### ISSUE TITLE
Filter title for games that use more that have special editions

### ISSUE THREAD

**Issue by JJAHAX2** [hulkrelax/hltb-for-deck#4]
For example it fails to find info for [Batman: Arkham Asylum GOTY Edition](https://store.steampowered.com/app/35140/Batman_Arkham_Asylum_Game_of_the_Year_Edition/) because the title contains "GOTY Edition". So I think it would be awesome if it would filter out words out of the title.

Some games that it fails to find due to this is:
[**Batman: Arkham Asylum GOTY Edition**](https://store.steampowered.com/app/35140/Batman_Arkham_Asylum_Game_of_the_Year_Edition/)
[**Batman: Arkham City GOTY**](https://store.steampowered.com/app/200260/Batman_Arkham_City__Game_of_the_Year_Edition/)
[**Crysis 2 Maximum Edition**](https://store.steampowered.com/app/108800/Crysis_2__Maximum_Edition/)
[**Mafia II (Classic)** ](https://store.steampowered.com/app/50130/Mafia_II_Classic/)
[**Shadowrun Hong Kong Extended Edition**](https://store.steampowered.com/app/346940/Shadowrun_Hong_Kong__Extended_Edition/)

**Comment by hulkrelax** [comment]
Yeah I figured this was inevitable. Unfortunately there isn't always guaranteed 1:1 from Steam to HLTB. I'll see what I can do. Thanks for concrete examples

**Comment by saynsh** [comment]
Came upon this today, when viewing Batman Arkham Asylum GOTY Edition, as @JJAHAX2 described.

One solution could be to have a list of keywords/phrases that can be stripped from the game name string in case the response from hltb yields a list of zero games. Then make a new call.

**Comment by Castlemark** [comment]
I've also encountered this issue with [Pathfinder: Wrath of the Righteous - Enhanced Edition](https://store.steampowered.com/app/1184370/Pathfinder_Wrath_of_the_Righteous__Enhanced_Edition/). As @saynsh said, I think the best solution is to add a list of keywords to filter while searching.
